fermipy/plotting.py

```diff
@@ -1,15 +1,7 @@
-from __future__ import absolute_import, division, print_function, \
-    unicode_literals
-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function
 import copy
 import os
-import matplotlib
-
-try:
-    os.environ['DISPLAY']
-except KeyError:
-    matplotlib.use('Agg')
-
 import matplotlib.pyplot as plt
```

Comment:
What's the motivation for removing this?  I've found it useful if you want plotting to generate plots on a machine without an X server (e.g. the nodes on our computing cluster at SLAC).  Is there another way to accomplish the same behavior?